src/apps/cli/internal/flags/generate/clients.flags.ts

```diff
@@ -0,0 +1,7 @@
+import { sharedFlags } from './sharedFlags';
+
+export const clientsFlags = () => {
+  return {
+    ...sharedFlags
```

Comment:
If my assumption is correct then client command only works with the new generator?

Reply:
yes

we know there is a contributor who wants to work on the first `sdk` template - then we will do a followup PR to enable it in CLI